### ISSUE TITLE
debootstrap failed error

### ISSUE THREAD

**Issue by peteruithoven** [elementary/houston#669]
I'm seeing build fail for [elementary/files](https://travis-ci.org/elementary/files/builds/445123661?utm_medium=notification&utm_source=email) and [elementary/appcenter](https://travis-ci.org/elementary/appcenter) with the following error:

```
Creating pbuilder environment...
I: Distribution is bionic.
I: Current time: Tue Oct 23 13:32:41 UTC 2018
I: pbuilder-time-stamp: 1540301561
I: Building the build environment
I: running debootstrap
/usr/sbin/debootstrap
E: specified keyring file (/etc/apt/trusted.gpg) not found
E: debootstrap failed
E: Tail of debootstrap.log:
tail: cannot open '/var/cache/liftoff-build//10/debootstrap/debootstrap.log' for reading: No such file or directory
W: Aborting with an error
I: cleaning the build env 
I: removing directory /var/cache/liftoff-build//10 and its subdirectories
```

**Comment by btkostner** [comment]
We know. We are working on it.

**Comment by btkostner** [comment]
Fixed in cf0069bbf10095a1a27acdff291aabdccd6b4c0d
